resources/language/resource.language.zh_tw/strings.po

```diff
@@ -0,0 +1,1092 @@
+msgid "The Kodi library has been updated"
+msgstr "Kodi資料庫已更新"
+
+msgctxt "#30221"
+msgid "Owner account"
+msgstr "擁有者帳戶"
+
+msgctxt "#30222"
+msgid "Kid account"
+msgstr "兒童帳戶"
+
+# Unused 30223
+msgctxt "#30224"
+msgid "Auto update mode"
+msgstr "自動更新模式"
+
+msgctxt "#30225"
+msgid "Manual"
+msgstr "手動"
+
+msgctxt "#30226"
+msgid "Scheduled"
+msgstr "計畫"
+
+msgctxt "#30227"
+msgid "Synchronize Kodi library with 'My list'"
+msgstr "將Kodi資料庫與我的清單同步"
+
+msgctxt "#30228"
+msgid "Set a profile for synchronization"
+msgstr "為同步設置一個設定檔"
+
+# Unused 30229
+msgctxt "#30230"
+msgid "Check for updates now"
+msgstr "立即檢查更新"
+
+msgctxt "#30231"
+msgid "Do you want to check update now?"
+msgstr "您要立即檢查更新嗎？"
+
+msgctxt "#30232"
+msgid "Profile maturity rating for {}"
+msgstr "{}的個人資料年齡分級"
+
+msgctxt "#30233"
+msgid "Only show titles rated [B]{}[/B]."
+msgstr "僅顯示評分為[B]{}[/B]的標題。"
+
+msgctxt "#30234"
+msgid "Install Up Next add-on"
+msgstr "安裝Up Next套件程式"
+
+msgctxt "#30235"
+msgid "Synchronize the watched status of the videos with Netflix"
+msgstr "將影片的觀看狀態與Netflix同步"
+
+msgctxt "#30236"
+msgid "Change watched status (locally)"
+msgstr "更改監視狀態（本地）"
+
+msgctxt "#30237"
+msgid "Marked as watched|Marked as unwatched|Restored Netflix watched status"
+msgstr "標記為已觀看|標記為未觀看|已恢復Netflix觀看狀態"
+
+msgctxt "#30238"
+msgid "Up Next notifications (watch next video)"
+msgstr "Up Next通知（觀看下一個影片）"
+
+msgctxt "#30239"
+msgid "Color of supplemental plot info"
+msgstr "補充資訊文字顏色"
+
+msgctxt "#30240"
+msgid "The background services cannot be started due to this problem:[CR]{}"
+msgstr "由於以下問題而無法啟動背景服務:[CR]{}"
+
+msgctxt "#30241"
+msgid "Open Connect CDN (the lower index is better)"
+msgstr "Open Connect CDN（索引越低越好）"
+
+msgctxt "#30242"
+msgid "Top 10"
+msgstr "十大排行"
+
+msgctxt "#30243"
+msgid "Select first unwatched TV show episode"
+msgstr "選擇第一個未觀看的電視節目影集"
+
+# Unused 30244
+msgctxt "#30245"
+msgid "Deleting files exported to the library"
+msgstr "刪除匯出到資料庫的內容"
+
+msgctxt "#30246"
+msgid "There are {} titles that cannot be imported.[CR]Do you want to delete these files?"
+msgstr "有{}標題無法導入，您是否要刪除這些內容"
+
+msgctxt "#30247"
+msgid "Results per page (in case of timeout error decrease it)"
+msgstr "每頁結果（請盡量請減少以避免逾時錯誤）"
+
+# Unused 30248 to 30299
+msgctxt "#30300"
+msgid "Do you want to remove watched status of '{}'?[CR][CR]The operation could take up to 24h, therefore the title may still appear in the list."
+msgstr "您是否要刪除已監視的'{}'狀態？[CR][CR]此操作最多可能需要24小時，因此標題可能仍顯示在清單中。"
+
+msgctxt "#30301"
+msgid "Include Kodi library (only sending data)"
+msgstr "包括Kodi資料庫（僅發送資料）"
+
+# Unused 30302 to 30339
+msgctxt "#30340"
+msgid "Choose the login method[CR](if the login with E-Mail/Password always returns \"incorrect password\", then use Authentication key, instructions in the GitHub ReadMe)"
+msgstr "選擇登入方式[CR]（如果使用電子郵件/密碼登入總是返回“不正確的密碼”，請使用身份驗證金鑰，查看GitHub讀我檔案中的說明）"
+
+msgctxt "#30341"
+msgid "E-Mail/Password"
+msgstr "電子信箱/密碼"
+
+msgctxt "#30342"
+msgid "Authentication key"
+msgstr "身份驗證金鑰"
```

Comment:
I think that this is better keep the english version,
the `Authentication key` is referred to the software/script name that generate this file
or do you think users understand anyway?

Reply:
OK, maybe keep "Authentication key" is more easily to find for user and not letting translate misconstrue